.gitignore

```
@@ -10,6 +10,7 @@ __diff_output__
 /.nyc_output
 /coverage
 /docs/.next
+/docs/pages/playground.tsx
```

This file was deleted, no?

```suggestion
/docs/pages/playground.example.tsx
```

---

I think the idea is to have `playground.example.tsx` on Github
But that if you create a `playground.tsx` on your machine, it should not be committed.

---

> But that if you create a playground.tsx on your machine, it should not be committed.

Then I have to create a new file every time I need to test something. It's easier to use `docs/pages/components/data-grid/demo.js` and not commit it.

---

Why would you need to create a new one every time ?

---

I have an alias that does `git restore -- . && git clean -f && git checkout next && git pull` to always have the latest changes from HEAD. I was thinking that it would delete the file but it didn't so I'm okay with pushing this forward.